tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "cookies" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "key" TEXT NULL,
+                    "value" TEXT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "session_data" (
+                    "sess_uuid" UUID NOT NULL PRIMARY KEY,
+                    "snare_uuid" TEXT NOT NULL,
```

Comment:
call this `sensor_id`. Call me an optimist, but we might have other sensors in the future using Tanner as a service ;)